achivement-service/src/main/resources/application-aws.yml

```
@@ -0,0 +1,59 @@
+          descriptor:
+            sql:
+              BasicBinder: debug
+      springframework:
+        transaction: debug
+        orm:
+          jpa: debug
+
+eureka:
+  client:
+    service-url:
+      defaultZone: ${EUREKA_SERVER}
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)